packages/camera/src/components/Capture/capture.js

```diff
@@ -415,7 +428,8 @@ const Capture = forwardRef(({
       onFinishUploadPicture={onFinishUploadPicture}
       addDamageParts={addDamageParts}
       onResetAddDamageStatus={handleResetDamageStatus}
-      hideAddDamage={hideAddDamage}
+      hideAddDamage={hideAddDamage || isPortraitModeVinLayoutView }
+      isPortraitModeVinLayoutView={isPortraitModeVinLayoutView}
```

Comment:
I think, you can pass isPortrait and selectedMode props directly here instead of isPortraitModeVinLayoutView prop and you can remove the same code in Layout component to get isPortrait value.